src/main/java/com/hotjava/app/hotjava/service/VehicleService.java

```diff
@@ -40,21 +40,30 @@ public VehicleService(IVehicleDAO vehicleDAO) {
     @Override
     @Cacheable(value="vehicle", key="#id")
     public Vehicle fetchById(int id) {
-        Vehicle foundVehicle = vehicleDAO.fetch(id);
-       return foundVehicle;
+       return vehicleDAO.fetch(id);
     }
 
     @Override
     @CacheEvict(value="vehicle", key="#id")
     public void delete(int id) throws Exception {
-        //vehicleDAO.delete(id);
+        try{
+            //vehicleDAO.delete(id);
+        }
+        catch (Exception e) {
```

Comment:
Exception handling for thrown exception.

Reply:
Ditto above.  An empty catch block is the worst thing to do.